redfish-core/lib/account_service.hpp

```diff
@@ -1638,10 +1639,22 @@ class ManagerAccount : public Node
     {
         if (password)
         {
-            if (!pamUpdatePassword(username, *password))
+            bool gotPamAuthtokError = false;
+            if (!pamUpdatePassword(username, *password, gotPamAuthtokError))
             {
                 BMCWEB_LOG_ERROR << "pamUpdatePassword Failed";
-                messages::internalError(asyncResp->res);
+                if (gotPamAuthtokError)
+                {
+                    messages::accountNotModified(asyncResp->res);
+                    // See https://redfishforum.com/thread/246/message-send-patch-password-failure
+                    asyncResp->res.jsonValue["error"]["Oem"]["IBM"] = "The "
```

Comment:
Did we make any changes in the schema(error_registry) to have error/Oem.IBM?

Reply:
I am unaware of any schema(error_registry).  I 'm not sure what you are asking about.

I see functions to add the AccountNotModified message ... but the changes in this PR are to add a *optional* Oem.IBM field to the AccountNotModified message, so I'm not sure where they would go, so I just put them into the code.  Is that what you were asking about?

I asked the Redfish forum the right way to do this here: https://redfishforum.com/thread/246/message-send-patch-password-failure

Reply:
Each standard message is having the registry schema, Here we are adding the oem.
So we need to change the schema. https://redfish.dmtf.org/registries/Base.1.5.0.json

Reply:
The way I understand the "Oem" property is this:
According to Redfish document DSP2046 (https://www.dmtf.org/sites/default/files/standards/documents/DSP2046_2019.1_0.pdf) the "Oem" property is already defined for all Redfish schemas.  Therefore, we can use it without having to update any Message Registry schemas.

Reply:
It might be better to change this code to use the AccountNotModified message Resolution field (see DSP2046 Message https://www.dmtf.org/sites/default/files/standards/documents/DSP2046_2019.2.pdf).  The resolution text could be something like:
PATCH the Password property again with a value that meets the password requirements.